src/CIG-Tests/CigPharoFunctionTemplateTest.class.st

```diff
@@ -116,5 +116,5 @@ CigPharoFunctionTemplateTest >> testGenerateMethodWithResultAsArgument [
 		assert: template generate
 		equals: 'getColor: color result: result
 
-	self ffiCall: #(void xx_TView_getColor(xx_TView* self, xx_ushort color, xx_TAttrPair* result))'
+	^ self ffiCall: #(void xx_TView_getColor(xx_TView* self, xx_ushort color, xx_TAttrPair* result))'
```

Comment:
But my fix (in CigPharoFunctionTemplate >> generate), also adds a ^ to this case which didn't need it (the C function returns void).